### PR TITLE
Feature reorient faces

### PR DESCRIPTION
**Pull request by bigfooted**
Fix face orientation of input mesh. Default is false. use --reorient or -t to activate
  
how to test: remesh a surface and then create a volume mesh from this surface:
$ pygalmesh-remesh-surface lion-head.off lionhead_remesh.off -e 0.025 -a 25 -s 0.1 -d 0.001
$ pygalmesh-volume-from-surface lionhead_remesh.off lionhead_remesh_vol.vtu
RuntimeError: Invalid input file "/tmp/tmpnq6ln74_.off"
If this is due to wrong face orientation, retry with the option --reorient "
$ pygalmesh-volume-from-surface lionhead_remesh.off lionhead_remesh_vol.vtu --reorient
$

### CODE REVIEW

**Comment by bigfooted** [comment]
The build fails, but I got the same thing when I tried to auto-install pygalmesh on my computer. I think it has to do something with the pip version. I had to manually install pygalmesh using python3 install

File: pygalmesh/main.py [hunk 1]
```diff
@@ -194,6 +194,7 @@ def generate_volume_mesh_from_surface_mesh(
     cell_radius_edge_ratio=0.0,
     cell_size=0.0,
     verbose=True,
+    reorient=True,
```
Comment (nschloe) [inline pygalmesh/main.py:197]: As discussed, default `False`. Also in the C++ below.

File: src/generate_from_off.cpp [hunk 2]
```diff
@@ -43,23 +50,51 @@ void generate_from_off(
...
+      msg << "Invalid input file \"" << infile << "\"" << std::endl;
+      msg << "If this is due to wrong face orientation, retry with the option --reorient \"" << std::endl;
+      throw std::runtime_error(msg.str());
+    }
   }
+
   input.close();
 
+  // NB: cgal has a check here to see if the input geometry is triangulated
+  // do we want it here?
```
Comment (nschloe) [inline src/generate_from_off.cpp:95]: Not sure. I'm not aware of any problems without the test.

**Comment by nschloe** [comment]
> The build fails, but I got the same thing when I tried to auto-install pygalmesh on my computer. I think it has to do something with the pip version. I had to manually install pygalmesh using python3 install

No, it passes. The failure is from gh-actions which doesn't have a recent-enough cgal. Ignore that.

**Comment by nschloe** [comment]
LGTM, thanks!

**Pull request merged**